beacon-chain/core/state/transition_fuzz_test.go

```diff
@@ -0,0 +1,226 @@
+package state
+
+import (
+	"context"
+	"testing"
+
+	fuzz "github.com/google/gofuzz"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	stateTrie "github.com/prysmaticlabs/prysm/beacon-chain/state"
+)
+
+func TestFuzzExecuteStateTransition_1000(t *testing.T) {
+	ctx := context.Background()
+	state := &stateTrie.BeaconState{}
+	sb := &ethpb.SignedBeaconBlock{}
+	fuzzer := fuzz.NewWithSeed(0)
+	fuzzer.NilChance(0.1)
+	for i := 0; i < 1000; i++ {
+		fuzzer.Fuzz(state)
+		fuzzer.Fuzz(sb)
+		s, err := ExecuteStateTransition(ctx, state, sb)
+		if err != nil {
```

Comment:
Please simplify to one if block.
```suggestion
		if err != nil && s != nil {
```

Here and everywhere in this file.

Reply:
thanks . simplified throughout the test file